code/Test_definitions/DeviceIdentifier_retrieve_identifier.feature

```diff
@@ -0,0 +1,148 @@
+@DeviceIdentifier_retrieve_identifier
+Feature: Camara Device Identifer API retrieve identifier
+
+# Input to be provided by the implementation to the tests
+# References to OAS spec schemas refer to schemas specified in code/API_definitions/CAMARA Mobile Device Identifier API.yaml
+# Implementation indications:
+# * api_root: API root of the server URL
+#
+# Testing assets:
+# * a mobile device with imei IMEI1 and phone number PHONENUMBER1
+# * a mobile device with imei IMEISV2  and phone number PHONENUMBER2
+# * a mobile device with DeviceIpv4Addr value IPV4ADDRESS
+
+@NumberVerification_verify
+Feature: Camara Number Verification API verify
+
+  # Input to be provided by the implementation to the tests
+  # References to OAS spec schemas refer to schemas specified in
+  # https://raw.githubusercontent.com/camaraproject/NumberVerification/main/code/API_definitions/number_verification.yaml
+  #
+  # Implementation indications:
+  # * api_root: API root of the server URL
+  #
+  # Testing assets:
+  # * a mobile device with SIM card with NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1
+  # * a mobile device with SIM card with NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER2
+  # * a mobile device with SIM card with NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER_HASHED1
+  # * a mobile device with SIM card with NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER_HASHED2
+
+
+  Background: Common Device Identifier retrieve identifer setup
+    Given the resource "/retrieve-identifier/v0"  as  base url
+    And the header "Content-Type" is set to "application/json"
+    And the header "Authorization" is set to a valid access token
+    And the header "x-correlator" is set to a UUID value
+    And the request body is compliant with the schema Device
+    And the header "x-correlator" is set to a UUID value
+    
+  @DeviceIdentifier_retrieve_identifier0_phoneNumber_does_not_match_schema
+  Scenario Outline: phoneNumber value does not comply with the schema
+    Given the request body property "$.phoneNumber" is set to: <phone_number_value>
+    When the HTTP "POST" request is sent
+    Then the response status code is 400
+    And the response property "$.status" is 400
+    And the response property "$.code" is "INVALID_ARGUMENT"
+    And the response property "$.message" contains a user friendly text
+    And they acquired a valid access token associated with PHONENUMBER1 through OIDC authorization code flow
```

Comment:
This test should be independent of how the access token was obtained. The API consumer should only really be explicitly specifying the `phoneNumber` if they are using a 2-legged token.